mkdocs.yml

```diff
@@ -34,6 +34,9 @@ nav:
       - Supported languages: status.md
       - Trophy case: trophy-case.md
   - Help:
+      - troubleshooting/app.md
```

Comment:
I suggest renaming `app.md` to `semgrep-app.md` just to keep closer to the naming convention.